modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/text/TextDDMFormFieldTemplateContextContributor.java

```diff
@@ -76,7 +77,21 @@
 				"hideField",
 				GetterUtil.getBoolean(ddmFormField.getProperty("hideField"))
 			).put(
-				"maxLength", ddmFormField.getProperty("maxLength")
+				"maxLength",
+				() -> {
+					try {
+						if (GetterUtil.getBoolean(
+								PropsUtil.get("feature.flag.LPS-146889"))) {
+
+							return null;
+						}
+
+						return ddmFormField.getProperty("maxLength");
+					}
+					catch (NullPointerException nullPointerException) {
+						return null;
+					}
```

Comment:
I didn't understand the need for this `catch`

Reply:
This is to avoid having to adapt unit tests because of a temporary feature flag. Some unit tests don't mock GetterUtil, so having this avoids NPE thrown by those tests.